nservicebus/upgrades/asp-6to1.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Azure Storage Persistence Upgrade Version 6 to 1
```

Comment:
6 to 1 is technically correct but looks like a typo. I don't have any better ideas, I'm just pointing it out :)